pkg/network/tracer/connection/ebpf_tracer.go

```diff
@@ -150,6 +152,8 @@ type ebpfTracer struct {
 
 	// periodically clean the ongoing connection pid map
 	ongoingConnectCleaner *ddebpf.MapCleaner[netebpf.SkpConn, netebpf.PidTs]
+	// periodically clean the enhanced TLS tags map
+	TLSTagsCleaner *ddebpf.MapCleaner[netebpf.ConnTuple, netebpf.TLSTagsWrapper]
```

Comment:
Do we actually need a cleaner?

Reply:
yes it will leak similarly to the `connection_protocol` map. the leak is slow but I was able to produce it in loadtests